comment_scan.rb

```diff
@@ -85,16 +85,34 @@ def restart(num_to_post, bundle)
             end
           when 'huh?'
             matched_regexes = report_raw(comment["post_type"], comment["body_markdown"])
-            if matched_regexes.empty?
-              cb.say "Comment didn't match any regexes", room_id
+            reason_text = ""
+            #Go through regexes we matched to build reason_text
+            matched_regexes.each { |regex_matched| reason_text += "Matched reason \"#{Reason.where(id: regex_matched["reason_id"])[0]["name"]}\" for regex: #{regex_matched["regex"]}\n" }
```

Comment:
For longish blocks like this, it's better to use the `do .. end` syntax and break it down as much as possible. Also, this could be something more like `reason_text = matched_regexes.map { |rm| "blah blah" }.join("\n")`